sdk/aqueduct/client.py

```diff
@@ -552,6 +560,8 @@ def publish_flow(
             source_flow:
                 Used to identify the source flow for this flow. This can be identified
                 via an object (Flow), name (str), or id (str or uuid).
+            use_local:
+                Used to identify if local data is used. Must be set if any artifact in the flow is derived from local data.
```

Comment:
nit: Let's scrape the first sentence. The second sentence is very clear already.